jupyter_sphinx/execute.py

```diff
@@ -443,14 +503,43 @@ def cell_output_to_nodes(cell, data_priority, dir):
     return to_add
 
 
-def attach_outputs(output_nodes, node):
-    if node.attributes['hide_code']:
-        node.children = []
-    if not node.attributes['hide_output']:
-        if node.attributes['code_below']:
-            node.children = output_nodes + node.children
-        else:
-            node.children = node.children + output_nodes
+def attach_outputs(output_nodes, node, thebe_config):
+    if thebe_config and not node.attributes['no_thebelab']:
+        code_class = 'thebelab-code'
+        if node.attributes['hide_code']:
+            code_class += ' thebelab-hidden'
+
+        code = node.astext()
+
+        node.children = [docutils.nodes.raw(
+            text='<pre class="{code_class}" data-executable="true" data-language="python">' \
+                '{data}' \
```

Comment:
Also, how is this rendered for writers other than HTML (does it degrade nicely?)
Perhaps we should use specific node classes rather than relying on `nodes.raw`.